Asaas/grails-app/services/asaas/PayerService.groovy

```diff
@@ -43,26 +43,35 @@ class PayerService {
     return payer
   }
 
-   private Map parseSaveParams(Map params) {
-      Map parsedParams = [:]
-
-      parsedParams.name = params.name
-      parsedParams.email = params.email
-      parsedParams.cpfCnpj = params.cpfCnpj
-      parsedParams.phone = params.phone
-      parsedParams.personType = params.personType instanceof String ? PersonType.convert(params.personType) : params.personType
-      parsedParams.street = params.street
-      parsedParams.number = params.number ? (params.number as Integer): null
-      parsedParams.neighborhood = params.neighborhood 
-      parsedParams.city = params.city
-      parsedParams.state = params.state
-      parsedParams.complement = params.complement
-      parsedParams.CEP = params.CEP
-      parsedParams.customerId = params.customerId ? (params.customerId as Long) : null
-
-      return parsedParams
-    }
+  public Payer getPayer(Long payerId){
+    Payer payer = Payer.read(payerId)
+    return payer
+  }
 
+  public List<Payer> list(Long customerId){
+    List<Payer> payerList = Payer.query(customerId: customerId).list()
+    return payerList
+  }
+
+  private Map parseSaveParams(Map params) {
+    Map parsedParams = [:]
+
+    parsedParams.name = params.name
+    parsedParams.email = params.email
+    parsedParams.cpfCnpj = params.cpfCnpj
+    parsedParams.phone = params.phone
+    parsedParams.personType = params.personType instanceof String ? PersonType.convert(params.personType) : params.personType
```

Comment:
se não for um PersonType mantém oque veio no params? Isso daria erro mais pra frente, convém tratar antes